windows/dirent_portable.h

```diff
@@ -1,7 +1,7 @@
 #ifndef SIMDBJSON_DIRENT_PORTABLE_INC_
 #define SIMDBJSON_DIRENT_PORTABLE_INC_
 
-#if (!defined(_WIN32) && !defined(_WIN64))
+#if (!defined(_WIN32) && !defined(_WIN64) && !(__MINGW32__) && !(__MINGW64__))
```

Comment:
Oh ... maybe dirent_portable will already do the job :)

Also, doh, just noticed SIMD**B**JSON up above.

Reply:
Fixing via another PR.